extensions-core/protobuf-extensions/pom.xml

```diff
@@ -36,6 +36,7 @@
 
   <properties>
     <commons-io.version>2.11.0</commons-io.version>
+    <okio.version>3.4.0</okio.version>
```

Comment:
any reason we use 3.4.0 here when the licenses.yaml indicates we use 3.6.0 in kubernetes-extensions? Can we align them?

Reply:
absolutely, I'll fix it.

Reply:
I wonder why okio doesn't show up in licenses.yaml for protobuf-extensions

Reply:
my money is on the scope - for protobuf it's provided, for kubernetes-extension it's compile

Reply:
if it is marked as provided, then I would not expect the okio jar to be included in the dependencies downloaded as part of the distribution packaging, and we can skip making any changes to this file.

Reply:
re-adding okio, as after the removal the integration tests failed. 
my suspicion is that given that protobuf-extensions expecst okio-2.8.0 provided, and we only have okio-3.6.0 in compile scope.